docker/container.go

```diff
@@ -622,11 +654,6 @@ func (c *Container) Log(ctx context.Context, follow bool) error {
 		return err
 	}
 
-	info, err := c.client.ContainerInspect(ctx, container.ID)
```

Comment:
This one is no more needed as `c.findExisting` already calls `ContainerInspect` :sweat_smile: